ui/src/ui/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,107 @@
+// Import packages
+import React, { useState } from 'react';
+import { toast } from 'react-hot-toast';
+
+// Import react icons
+import { FaUserCircle } from 'react-icons/fa';
+import { BsFillInfoCircleFill } from 'react-icons/bs';
+import { AiOutlineLogout } from 'react-icons/ai';
+
+// Import css
+import './Navbar.css';
+
+const Navbar = () => {
+  const [showProfile, setShowProfile] = useState(false);
+
+  const user: { email: string; displayName: string; profilePicture?: string } =
+    {
+      email: 'afaq.ahmed@xgrid.co',
+      displayName: 'Afaq Ahmed',
+    };
+
+  /**
+   * Function logout user
+   */
+  const onLogout = async () => {
+    toast.success('You are logged out successfully', {
+      duration: 3000,
+      position: 'bottom-center',
+      ariaProps: {
+        'role': 'status',
+        'aria-live': 'polite',
+      },
+    });
+  };
```

Comment:
You can create a separate component that displays a toast on the given properties
Properties could be message and error or success toast

Reply:
Done

Reply:
Done